core/backend/src/test/ecsql/queries/DerivedProperty.ecsql.md

```diff
@@ -37,11 +37,11 @@ LIMIT
 
 | ECInstanceId | Model                                  | dt                      | b    | bin                                | DirectStr | DirectLong | DirectDouble | p2d                         | p3d                                      |
 | ------------ | -------------------------------------- | ----------------------- | ---- | ---------------------------------- | --------- | ---------- | ------------ | --------------------------- | ---------------------------------------- |
-| 0x14         | {"Id": "0x11", "RelECClassId": "0x61"} | 2017-01-01T00:00:00.000 | true | BIN(1,2,3)                         | str0      | 1000       | 0.1          | {"X": 1.034,"Y": 2.034}     | {"X": -1,"Y": 2.3,"Z": 3.0001}           |
```

Comment:
Hm this is a navigation property, I don't think there is a nice way to make this schema agnostic. I suggest we remove the model column from this test